### ISSUE TITLE
[Bug] Console Error: `Warning: Extra attributes from the server: class`

### ISSUE THREAD

**Issue by atgctg** [OhMyGuus/I-Still-Dont-Care-About-Cookies#2283]
![Screenshot 2023-02-13 at 9 25 28 AM](https://user-images.githubusercontent.com/105969161/218407141-f9276c21-0576-4635-ab21-8baabcac30e4.png)

The extension seems to be interfering with my local Next.js development server running on port 3000. Enabling the extension causes error messages in my console (see above image) related to port 3000, and disabling the extension makes those errors go away.

Would it be possible to add a configuration or blocklist option to ignore specific local host URLs (e.g. localhost:3000) so that the extension does not affect local development servers?

**Website URL(s)**

- localhost:3000

**Browser**
[ ] Firefox
[x] Chrome

**Extension version**
`1.1.0`

**Comment by OhMyGuus** [comment]
Yeah, in my opinion localhost and 127.0.0.1 should be disabled as well.
